docs/reference/buildx_bake.md

```diff
@@ -50,6 +50,66 @@ guide for introduction to writing bake files.
 
 ## Examples
 
+### <a name="allow"></a> Allow extra privileged entitelement (--allow)
+
```

Comment:
Would be good to explain here a bit why allowing entitlements is needed and in what cases it protects the user.